### DISCUSSION TITLE
启用自定义顶栏后点击logo会导致打开新标签页

### DISCUSSION THREAD

**Discussion by xslingcn** [the1812/Bilibili-Evolved#2051] [综合讨论 / General]
开启自定义顶栏后logo处后变成：
`<a href="https://www.bilibili.com/" target="_blank" class="main-content"><img height="38" class="custom-navbar-season-logo" src="https://i0.hdslb.com/bfs/feed-admin/10641bbc5189591221c00958f3458f33798c7caa.png"></a>`
页内跳转或许更符合使用习惯，希望能进行更改或增加设置选项。

**Comment by Geeyun-JY3** [comment]
源自：#570

**Reply by xslingcn** [reply]
啊……
左键单击在当前标签页跳转，ctrl加左键在新标签页打开，shift加左键在新窗口打开，这不应该是约定俗成的用户习惯吗。
况且当前主流的网页设计中点击navbar logo的含义一般还是为页内跳转回首页。

**Reply by Geeyun-JY3** [reply]
我就给个上下文。我也支持你这个 FR，我也希望能给个设置项。

**Reply by the1812** [reply]
> 左键单击在当前标签页跳转，ctrl加左键在新标签页打开，shift加左键在新窗口打开，这不应该是约定俗成的用户习惯吗

我也是这习惯, 本来代码也是这么写的, 不过人多了无论新不新标签页总有人说不习惯的()
不过还是跟原版对齐下比较好

**Reply by Geeyun-JY3** [reply]
同意遵照原版。这样也就能把锅都甩给 B 站啦（bushi
